openfermioncirq/gates/fermionic_simulation.py

```diff
@@ -87,6 +88,90 @@ def state_swap_eigen_component(x: str, y: str, sign: int = 1, angle: float = 0):
     return component
 
 
+def fermionic_simulation_gates_from_interaction_operator(
+        operator: openfermion.InteractionOperator):
+    r"""
+    Given $H = \sum_a H_a$, returns gates
+    $\left\{G_a\right\} = \left\{e^{i H_a\right\}$.
```

Comment:
This docstring needs more detail. It's not clear how the Hamiltonian is being decomposed into the sum (there are multiple ways of grouping the components of an InteractionOperator).

Reply:
Done.

Do you know why the docstring doesn't show up in the built docs? This seems to be the case for a lot of functions in OFC, e.g. [ffft](https://openfermion-cirq.readthedocs.io/en/latest/generated/openfermioncirq.ffft.html).